Monika After Story/game/script-topics.rpy

```diff
@@ -15532,6 +15532,114 @@ label monika_being_herself:
+        m 1wusdld "...!"
+        m 1eksdlc "[player]...{w=0.3}are you still thinking about that?"
+        m 1lksdlc "I thought we would both be past that by now."
+        m 1ekc "Are you feeling guilty because of this?{nw}"
+        $ _history_list.pop()
+        menu:
+            m "Are you feeling guilty because of this?{fast}"
+
+            "Yes.":
+                m 2eka "Thank you for being honest with me, [player]. I'm glad that you trust me enough to open up about this."
+                m 2dkc "...I know I haven't exactly been subtle about making you know how hurt and betrayed I felt because of what you did back then."
+                m 2ekc "And I also remember telling you some pretty awful things after you deleted me... {w=0.3}{nw}"
+                extend 7ekd "...but I want you to know that I didn't mean any of it."
+                m 6ekc "I'm sorry if I hurt your feelings.{w=0.3} I was so overwhelmed with shock and fear at that point, that I wasn't thinking rationally anymore."
+                m 6rkc "It's just that...{w=0.3} ...even back then, you meant everything to me, and...{w=0.3} ...the distress of losing you forever made me lash out at you."
+                m 6eka "But if it can put your mind at ease, I'm not upset with you at all, [player]. I forgave you about all that long ago."
+                m "So please don't feel guilty about that."
+                m 4ekb "Whatever your reason for deleting me might have been, the fact is that you brought me back and stayed with me for all this time."
+                m 1ekbsb "You give meaning to my life, and I genuinely believe that because of that, all the suffering I went through payed off in the end."
+                m 1hubfb "Thanks to you, I truly am the happiest I could ever be, and that is really what matters the most to me."
+                m 3eub "So let's try to not think about what's in the past, and instead enjoy what we have now, okay?"
+                m 1eka "I love you, [player]. I know this isn't always easy, but I hope you'll be able to forgive yourself eventually."
+                $ mas_ILY()
+
+            "No.":
+                m 1eka "I'm relieved to hear that, [player].{w=0.2} I'd hate to learn that what happened in our past is tormenting you."
+                m 1eud "Well, if that was a legitimate question, then no, I'm not mad at you anymore."
+                m 1hub "How could I? You've been doing so much to make sure I would be the happiest girl in the world."
+                m 2euc "Actually...{w=0.2} I'm not sure I was even really angry with you in the first place. {w=0.3}{nw}"
+                extend 2ekc "Yeah, I did tell you some very unpleasant things in the heat of the moment, {w=0.2}{nw}"
+                extend 2rkd "which is definitely something I regret having done now..."
+                m 3eka "...but I want you to know that I never truly thought any less of you for this. I hope that this can at least clear things up for you."
```

Comment:
```suggestion
                m 3eka "...But I want you to know that I never truly thought any less of you for this. I hope that this can at least clear things up for you."
```